javascript/javascript1/week3/series-duration.js

```diff
@@ -0,0 +1,59 @@
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "Sopranos",
+        days: 3,
+        hours: 14,
+        minutes: 0,
+    },
+    {
+        title: "The Wire",
+        days: 2,
+        hours: 12,
+        minutes: 0,
+    },
+    {
+        title: "Friends",
+        days: 4,
+        hours: 2,
+        minutes: 20,
+    },
+    {
+        title: "How i met your mother",
+        days: 7,
+        hours: 6,
+        minutes: 36,
+    },
+    {
+        title: "House of cards",
+        days: 7,
+        hours: 10,
+        minutes: 16,
+    },
+    {
+        title: "Money heist",
+        days: 8,
+        hours: 22,
+        minutes: 26,
+    },
+];
+const totalMinutes = 365.25 * 80 * 24 * 60;
+
+function CalAvg(seriesDurations) {
```

Comment:
a function name should start with a lowercase.

Also I think the function was supposed to be named `logOutSeriesText` (according to [the homework](https://github.com/HackYourFuture-CPH/JavaScript/blob/master/javascript1/week3/homework.md#series-duration-of-my-life)).

Reply:
fix the fun name.